### PR TITLE
add perk slave trader from roguery

### PR DESCRIPTION
**Pull request by miguelcjalmeida**
Adds the roguery perk 'Slave trader' which increases your profits from ransoming by 20%. It seems every ransom related action calls the method CharacterObject.PrisonerRansomValue to determine the value of the prisioner; even when bartering them.

### CODE REVIEW

File: src/CommunityPatch/Patches/Perks/Cunning/Roguery/SlaveTraderPatch.cs [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+using HarmonyLib;
+using TaleWorlds.CampaignSystem;
+using TaleWorlds.Core;
+using TaleWorlds.Localization;
+using static CommunityPatch.HarmonyHelpers;
+
+namespace CommunityPatch.Patches.Perks.Cunning.Roguery {
+
+  public sealed class SlaveTraderPatch : PatchBase<SlaveTraderPatch> {
+
```
Comment (iPherian) [inline src/CommunityPatch/Patches/Perks/Cunning/Roguery/SlaveTraderPatch.cs:13]:
```suggestion
    private const float BonusRansomGoldPercent = .20f;

```

File: src/CommunityPatch/Patches/Perks/Cunning/Roguery/SlaveTraderPatch.cs [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+        }
+      );
+
+      _perk.Initialize(
+        textObjStrings[0],
+        textObjStrings[1],
+        _perk.Skill,
+        (int) _perk.RequiredSkillValue,
+        _perk.AlternativePerk,
+        _perk.PrimaryRole, .20f,
```
Comment (iPherian) [inline src/CommunityPatch/Patches/Perks/Cunning/Roguery/SlaveTraderPatch.cs:62]:
```suggestion
        _perk.PrimaryRole, BonusRansomGoldPercent,
```

File: src/CommunityPatch/Patches/Perks/Cunning/Roguery/SlaveTraderPatch.cs (hunk 1 above)
Comment (Tyler-IN) [reply src/CommunityPatch/Patches/Perks/Cunning/Roguery/SlaveTraderPatch.cs:13]: I think we can do this pattern in bulk to many perks in a refactor PR later.

File: src/CommunityPatch/Patches/Perks/Cunning/Roguery/SlaveTraderPatch.cs (hunk 2 above)
Comment (Tyler-IN) [reply src/CommunityPatch/Patches/Perks/Cunning/Roguery/SlaveTraderPatch.cs:62]: I think we can do this pattern in bulk to many perks in a refactor PR later.

**Pull request merged**